src/Controller/AdminResettingController.php

```diff
@@ -186,15 +186,31 @@ private function sendResettingEmailMessage(UserInterface $user): void
             'confirmationUrl' => $url,
         ]);
 
+        $this->sendEmailMessage(
+            $rendered,
+            $this->container->getParameter('fos_user.resetting.email.from_email'),
+            (string) $user->getEmail()
+        );
+    }
+
+    /**
+     * @param string       $renderedTemplate
+     * @param array|string $fromEmail
+     * @param array|string $toEmail
+     */
+    private function sendEmailMessage($renderedTemplate, $fromEmail, $toEmail): void
+    {
         // Render the email, use the first line as the subject, and the rest as the body
-        $renderedLines = explode(PHP_EOL, trim($rendered));
+        $renderedLines = preg_split('/\R/', trim($renderedTemplate), 2, PREG_SPLIT_NO_EMPTY);
```

Comment:
This looks wrong. Why would you have different line encodings inside your application?

Reply:
Simple test to illustrate my issue on Windows, because PHP_EOL is "\r\n" there.
```php
    public function testEOL()
    {
        $template = "Subject\nBody";

        $renderedLines = explode(PHP_EOL, trim($template));
        $subject = array_shift($renderedLines);
        $body = implode(PHP_EOL, $renderedLines);

        $this->assertEquals('Subject', $subject);
        $this->assertEquals('Body', $body);
    }
```
```
1) x::testEOL
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'Subject'
+'Subject
+Body'
```